.travis.yml

```diff
@@ -67,3 +67,5 @@ jobs:
       env: PLATFORM=icl
     - <<: *qemuboottest
       env: PLATFORM=imx8
+    - <<: *qemuboottest
+      env: PLATFORM=imx8x
```

Comment:
Who's the actual author of this commit? I don't like seeing that the commit author and the sign-off are different.

Reply:
@dianacretu something must have gone wrong when splitting the patch. Please fix the author for this patch :).